Test/Microsoft.ApplicationInsights.Test/Shared/Extensibility/Implementation/Tracing/HeartbeatTests.cs

```diff
@@ -173,16 +257,20 @@ public void HeartbeatPayloadContainsUserSpecifiedData()
         [TestMethod]
         public void HeartbeatPayloadContainsOnlyAllowedDefaultPayloadFields()
         {
+            var assemblyDefFields = new BaseDefaultHeartbeatPropertyProvider();
```

Comment:
What is this test testing? No asserts found .

Reply:
There is an assert on line 285. Testing that explicitly removed or 'disabled' properties are not in the heartbeat.